gitbranchstack/main.py

```diff
@@ -285,8 +285,16 @@ def on_conflict(path):
         global ON_CONFLICT
         ON_CONFLICT = on_conflict
         head = rebase(patch, head)
+        message = head.message
         if t == topic or trim_all_subjects or trim_subject:
-            head = head.update(message=trimmed_message(subject, patch.message))
+            message = trimmed_message(subject, patch.message)
+        head = repo.new_commit(
+            message=message,
+            tree=head.tree(),
+            parents=head.parents(),
+            author=head.author,
+            committer=patch.committer, # preserve original author and timestamp
```

Comment:
I guess

```suggestion
            committer=patch.committer, # preserve original committer and timestamp
```

I agree that technically we should use the current user's as committer name/email (and that it's low prio).

Reply:
Yeah, good call :+1: